azure-pipelines.yml

```diff
@@ -66,50 +66,48 @@ jobs:
       python ./scripts/ci/test_index.py -v
     displayName: "Verify Extensions Index"
 
-# - job: TestsPython27
-#   displayName: "Tests Python 2.7"
-#   pool:
-#     vmImage: 'ubuntu-16.04'
-#   steps:
-#   - task: UsePythonVersion@0
-#     displayName: 'Use Python 2.7'
-#     inputs:
-#       versionSpec: 2.7
-#   - task: Bash@3
-#     displayName: "Tests Python 2.7"
-#     inputs:
-#       targetType: 'filePath'
-#       filePath: scripts/ci/test_source.sh
+- job: SourceTests
+  displayName: "Integration Tests, Build Tests"
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
```

Comment:
I'm curious why we support 3.6 and 3.8 here, while support 3.7 in other jobs

Reply:
Python 3.6 takes 57%+ market share, Python 3.7 takes 10%- https://w3techs.com/technologies/details/pl-python/3